js/electron.js

```diff
@@ -47,7 +47,7 @@ function createWindow() {
 
 	// and load the index.html of the app.
 	//mainWindow.loadURL('file://' + __dirname + '../../index.html');
-	mainWindow.loadURL("http://localhost:" + config.port);
+	mainWindow.loadURL(`http://${config.address}:${config.port}`);
```

Comment:
No, you're right, templating is the right thing to do, didnt notice that (but I am of course not the maintainer to inforce any other rule)